docs/parameters.md

```diff
@@ -55,9 +55,9 @@ https://github.com/kircherlab/CADD-scripts/#manual-installation.</small></detail
 | `snv_call_regions` | A BED file containing regions to limit SNV calling. By default this is the same as `target_regions`. | `string` |  |  |  |
 | `sv_call_regions` | A BED file containging regions to filter SV calls. By default this is the same as `target_regions`. | `string` |  |  |  |
 | `qc_regions` | A BED file with regions of interest used in QC. By default this is the same as `target_regions`. | `string` |  |  |  |
-| `hificnv_expected_xy_cn` | A BED file containing expected copy number regions for XY samples. | `string` |  |  |  |
-| `hificnv_expected_xx_cn` | A BED file containing expected copy number regions for XX samples. | `string` |  |  |  |
-| `hificnv_excluded_regions` | A BED file specifying regions to exclude with HiFiCNV, such as centromeres. | `string` |  |  |  |
+| `cnv_expected_xy_cn` | A BED file containing expected copy number regions for XY samples. | `string` |  |  |  |
+| `cnv_expected_xx_cn` | A BED file containing expected copy number regions for XX samples. | `string` |  |  |  |
+| `cnv_excluded_regions` | A BED file specifying regions to exclude with HiFiCNV, such as centromeres. | `string` |  |  |  |
```

Comment:
Maybe we should write somewhere where you can acquire these files or are they part of the nallorefs download

Reply:
Ah missed that part. I think it is enough as you have it